edu.gemini.lch.model/src/main/java/edu/gemini/lch/model/BaseLaserNight.java

```diff
@@ -37,57 +34,52 @@ public Site getSite() {
         return site;
     }
 
-    public DateTime getStart() {
-        return new DateTime(start);
+    public ZonedDateTime getStart() {
+        return ZonedDateTime.ofInstant(start, ZoneId.systemDefault());
```

Comment:
Are you explicitly setting the system default at launch time or are you relying on the OS?

Reply:
For joda `DateTimeZone`:

    * The default time zone is derived from the system property {@code user.timezone}.
     * If that is {@code null} or is not a valid identifier, then the value of the
     * JDK {@code TimeZone} default is converted. If that fails, {@code UTC} is used.
     * <p>
     * NOTE: If the {@code java.util.TimeZone} default is updated <i>after</i> calling this
     * method, then the change will not be picked up here.

For Java 8 `ZoneId`:
     * Gets the system default time-zone. (`java.util.TimeZone`)
     * This queries `TimeZone.getDefault()` to find the default time-zone
     * and converts it to a `ZoneId`. If the system default time-zone is changed,
     * then the result of this method will also change. (NOTE THAT THAT THIS DIFFERS FROM JODA, which is why I avoid almost all use of `TimeZone` in the code unless absolutely necessary).

I don't see any explicit setting of the the system default at launch time, so since they both query the system default time-zone, I did not add this. We should in the future, though, and set it to UTC, I think, to make things crystal clear, and then only switch to local date times when necessary for view purposes, as you proposed, I think.

Reply:
You can do
`java -Duser.timezone=America/New_York `